src/fu-util.c

```diff
@@ -2815,6 +2819,8 @@ main (int argc, char *argv[])
 		priv->flags |= FWUPD_INSTALL_FLAG_FORCE;
 	if (no_history)
 		priv->flags |= FWUPD_INSTALL_FLAG_NO_HISTORY;
+	if (simulate)
+		priv->flags |= FWUPD_INSTALL_FLAG_SIMULATE;
```

Comment:
Is there actually value in this going into fwupdmgr?  It feels like a developer feature, it should probably only be in fwupdtool.

Reply:
I guess it depends if the behaviour is expected to be identical between fwupdtool and fwupdmgr -- in the single device case that should be true, but I'm not sure about the composite case.

Reply:
A while back yes - but the composite case I was expecting to be true as well.  We explicitly moved a lot of code into the daemon to make them behave identically.  If there actually is a case it's not that this will help root out, then by all means lets leave it in both.

Reply:
I also wondered if we should expose the simulation as part of a "test transaction" in gnome-firmware for instance.